Patches/Harpy/Patch_Prosthetics_Harpy.xml

```diff
@@ -0,0 +1,158 @@
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>plasteel harpy claw</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>18</power>
+							<cooldownTime>0.93</cooldownTime>
+							<armorPenetrationSharp>5.5</armorPenetrationSharp>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>plasteel harpy claw</label>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<power>12</power>
+							<cooldownTime>0.93</cooldownTime>
+							<armorPenetrationSharp>7.5</armorPenetrationSharp>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Mono Claw -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="HarpyMonoClaw"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy monoclaw</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>26</power>
+							<cooldownTime>0.78</cooldownTime>
+							<armorPenetrationSharp>12</armorPenetrationSharp>
+							<armorPenetrationBlunt>8</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy monoclaw</label>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<power>14</power>
+							<cooldownTime>0.78</cooldownTime>
+							<armorPenetrationSharp>16</armorPenetrationSharp>
+							<armorPenetrationBlunt>8</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Venom Claw -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="HarpyVenomClaw"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy venonmclaw</label>
+							<capacities>
+								<li>ScratchToxic</li>
+							</capacities>
+							<power>21</power>
+							<cooldownTime>0.76</cooldownTime>
+							<armorPenetrationSharp>5.5</armorPenetrationSharp>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>harpy venonmclaw</label>
+							<capacities>
+								<li>ScratchToxic</li>
+							</capacities>
+							<power>12</power>
+							<cooldownTime>0.76</cooldownTime>
+							<armorPenetrationSharp>7.5</armorPenetrationSharp>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+						</li>
+					</tools>
+				</value>
+			</li>
+			
+			<!-- Work Claw -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/HediffDef[defName="HarpyWorkClaw"]/comps/li[@Class="HediffCompProperties_VerbGiver"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">

```

Comment:
Since this a "work claw," I assume it's more suited for work than combat? I'd reduce the damage to about the same as the steel claw, but reduce the sharp penetration by half.